src/extension.ts

```diff
@@ -163,83 +165,54 @@ export function registerFileWatcher(projectProvider: ProjectProvider): void {
 	});
 }
 
-function startupLibertyLanguageServer(context: ExtensionContext) {
-    //Start up Liberty Language Server
-    const languageServerPath = context.asAbsolutePath(path.join("jars", LIBERTY_LS_JAR));
-    // Language server options 
-    const serverOptions: ServerOptions = {
-        run: <Executable> { 
-            command: "java",
-            args: ["-jar", languageServerPath],
-            options: {stdio:"pipe"}
-        },
-        debug: <Executable> {
-            command: "java",
-            // TODO: using the debug arguments seems to still run the language server and open the debug port,
-            // but, the extension doesn't seem to work properly nor does it run the .onReady.then() commands
-            // const debugArgs = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=localhost:1074";
-            // args: [debugArgs, "-jar", languageServerPath],
-            args: ["-jar", languageServerPath],
-            options: {stdio:"pipe"}
-        }
-    };
-
-    // Options to control the language client
-    const clientOptions: LanguageClientOptions = {
-        documentSelector: SUPPORTED_LANGUAGE_IDS,
-        synchronize: {
-            configurationSection: ["properties", "plaintext"],
-            fileEvents: [
-                workspace.createFileSystemWatcher("**/bootstrap.properties"),
-                workspace.createFileSystemWatcher("**/server.env")
-            ],
-        }
-    };
-
-    console.log("Creating new language client for Liberty Language Server");
-    libertyClient = new LanguageClient(LIBERTY_CLIENT_ID, "Language Support for Liberty", serverOptions, clientOptions);
-    
-    const disposable = libertyClient.start();
-    context.subscriptions.push(disposable);
-
-    return libertyClient.onReady();
+function startupLibertyLanguageServer(context: ExtensionContext, api: JavaExtensionAPI) {
+    return resolveRequirements(api).then(requirements => {
+        //Start up Liberty Language Server
+
+        // Options to control the language client
+        const clientOptions: LanguageClientOptions = {
+            documentSelector: SUPPORTED_LANGUAGE_IDS,
+            synchronize: {
+                configurationSection: ["properties", "plaintext"],
+                fileEvents: [
+                    workspace.createFileSystemWatcher("**/bootstrap.properties"),
+                    workspace.createFileSystemWatcher("**/server.env")
+                ],
+            }
+        };
+
+        const serverOptions = prepareExecutable(LIBERTY_LS_JAR, requirements)
+
+        console.log("Creating new language client for Liberty Language Server");
+        libertyClient = new LanguageClient(LIBERTY_CLIENT_ID, "Language Support for Liberty", serverOptions, clientOptions);
```

Comment:
I believe this is the text that appears in the "Output" tab correct? If so we should localize this message